### DISCUSSION TITLE
Is this project still actively maintained ?

### DISCUSSION THREAD

**Discussion by eric-edouard** [keystonejs/keystone#9385] [Questions] [answered]
Loving it so far, and I understand it has reached maturity for production use.
But although the roadmap seems ambitious, I'm seeing that there isn't much activity anymore.
The latest commits are mostly from dependency bots.
I'm asking because even though the experience is amazing to quickly spin a backend + adminUI, I'm not sure how future proof it is.
Thanks

**Comment by dcousens** [comment, marked as answer]
Hi @eric-edouard,

We [last updated](https://github.com/keystonejs/keystone/pull/9251) our website roadmap in August, and are actively maintaining the project and accepting community contributions.  Please see are last release in [August](https://github.com/keystonejs/keystone/releases/tag/2024-08-28).

Development isn't necessarily as fast as the [community wants](https://github.com/keystonejs/keystone/pull/9253#issuecomment-2378277810) (and @gautamsi, I hear you :yellow_heart:) but, to your question @eric-edouard, yes this project is actively maintained.

Dan
